### DISCUSSION TITLE
Sidekick20 and catridge loading problems

### DISCUSSION THREAD

**Discussion by OldWer** [frntc/Sidekick64#103] [Ideas]
I tried loading cartridges with multiple files, but multiple selection does not work for me - the space key does nothing in the browser. I wonder what I am doing wrong?

Then I found out I could make .crt files that Sidekick20 recognizes, using the cartconv.exe tool from the Vice emulator. When I convert multiple catridge files in this way, they load OK, and some play OK, but many of them crash to basic or exhibit other strange behavior. I think this must be Sidekick20's fault, because they run fine in Vice. Is there any chance for a better cartridge support?

**Comment by leofoe** [comment]
I have the same problem. Did you manage to solve this problem?

**Comment by leofoe** [comment]
I managed to get some TOSEC cartridges to work with the following conversion:
`cartconv -l 40960 -t vic20 -i "Choplift.crt" -o "Choplifter.crt"`
Note the using of the -l switch, which designates the start address (in this case 40960 or hex A000).
